src/translations/qterminal_de.ts

```diff
@@ -270,17 +270,17 @@
     <message>
         <location filename="../mainwindow.cpp" line="449"/>
         <source>&amp;BlockCursor</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Blockschreibmarke</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="450"/>
         <source>&amp;UnderlineCursor</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Unterstrichschreibmarke</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="451"/>
         <source>&amp;IBeamCursor</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Vertikalschreibmarke</translation>
```

Comment:
dito - vertikaler Cursor - Cursor ist soweit eingedeutscht - kein Mensch weiss, was eine Schreibmarkenbewegungstaste ist - ne Cursor-Taste kennen die Leute. (Doofes Beispiel - übersetz mal Motor ohne Fremdworte - Verbrennungskraftmaschine - und da hängste wegen der Maschine wieder in der Uhr:
Eine Maschine (entlehnt aus französisch machine, von lateinisch machina, dieses von altgriechisch μηχανή mēchanḗ „Werkzeug, künstliche Vorrichtung, Mittel“[1][2]) ist ein technisches Gebilde mit durch ein Antriebssystem bewegten Teilen.